### ISSUE TITLE
close tab on logout from ssh

### ISSUE THREAD

**Issue by somepad** [noodle1983/putty-nd3.x#2]
When pressing Ctrld-D and logging out from SSH session, session tab must close as it was in previous versions.
How do you think?

**Comment by jozuman** [comment]
I don't agree. I think it is better how it is with the tab remaining open even though the session was closed.

**Comment by somepad** [comment]
If you closed the session - why would you need it's tab to be open?
If you have lost the session (e.g. by network issues) - tab should not close.
